src/dbnode/storage/database.go

```diff
@@ -548,12 +548,12 @@ func (d *db) Write(
 		return err
 	}
 
-	series, err := n.Write(ctx, id, timestamp, value, unit, annotation)
+	series, shouldWrite, err := n.Write(ctx, id, timestamp, value, unit, annotation)
```

Comment:
I think this makes more sense as `wasWritten` instead of `shouldWrite`. It makes more sense to me as the buffer returning a bool indicating whether it wrote or not as opposed to the buffer telling the commitlog what to do

Reply:
Sure, sounds good to me